src/AnalyticsEventReporter.ts

```diff
@@ -34,10 +34,27 @@ export class AnalyticsEventReporter implements AnalyticsEventService {
   with(payload: EventPayload): AnalyticsEventService {
     const currentPayload =
       this.payload === undefined ? payload : merge(this.payload, payload);
+    if (this.config.debug) {
+      console.log('[DEBUG] AnalyticsConfig Object:\n', this.config);
```

Comment:
Try using template literals. This might help with the formatting and readability of the strings

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals